tools/cucumber.puml

```diff
@@ -0,0 +1,64 @@
+@startuml
+sprite $cucumber [64x52/16] {
+0000000000000000000000000000000000000000000000000000000000000000
```

Comment:
The low contrast between the speech bubble gray and the text ".js" makes it bad to read/recognize. Should we enhance the contrast a little bit?

Also we should see to it that all images have the same dimensions, that is 64x64 px.

![grafik](https://user-images.githubusercontent.com/6781173/117305188-cd651c00-ae7e-11eb-8574-5ad6b5f12de4.png)

Reply:
I can try to tweak the contrast. About the dimension of the sprites: In the readme it says 64x64px but many of the other sprites are in 48x48px what is correct here?